### PR TITLE
Adds a run.sh for quick testing under AEMU

### PR DESCRIPTION
**Pull request by prologic**
Also adds a screenshot of this working.

### CODE REVIEW

**Comment by ailispaw** [comment]
`script` directory goes to the container to build barge.
So `contrib/qemu/` might be a good place for it to be and also we need to put the `contrib` into `.dockerignore`.
Could you move the screenshot to the directory and make `REAME.md` accordingly there?

**Comment by ailispaw** [comment]
And FYI: https://github.com/bargees/barge-packer/tree/master/qemu

**Comment by prologic** [comment]
Done

**Comment by prologic** [comment]
> And FYI: https://github.com/bargees/barge-packer/tree/master/qemu

Hmm interesting didn't see  this and not familiar with Packer

**Comment by ailispaw** [comment]
I meant that the screenshot in `REAME.md` and the image itself should be the same directory `contrib/qemu/`.
And cold you make another `REAME.md` there for qemu and the screenshot instead of the main `README.md`?

File: contrib/qemu/run.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+echo
+
+if [ -n "$bridge" ]; then
+  network=-net bridge,br=br0 -net nic
+else
+  network=""
+fi
+
+if (( gui == 1 )); then
+  qemu-system-x86_64 -m 1G -cdrom output/barge.iso $network
```
Comment (ailispaw) [inline contrib/qemu/run.sh:28]: `output/barge.iso` to `../../output/barge.iso` or `barge.iso` for example?

**Comment by ailispaw** [comment]
You may need virtual HDD and port-forwarding like this.  
https://github.com/bargees/barge-os/issues/75#issuecomment-381807749

**Comment by prologic** [comment]
Removed the screenshot; it doesn't add much value if it'll be in a separate README.

**Comment by prologic** [comment]
> You may need virtual HDD and port-forwarding like this.

Not necessary IHMO as this is only for quick boot-up of the built images and testing. I don't see this as *real production usage* at all.

**Comment by ailispaw** [comment]
Oh now I understand that it's for testing but an example.

**Comment by prologic** [comment]
Pretty much ;) I borrowed it from vallinux's scripts :)

**Comment by ailispaw** [comment]
> Removed the screenshot; it doesn't add much value if it'll be in a separate README.

I don't think so, even it's just a test's result. I like it.

**Comment by prologic** [comment]
Fine but I won't put it in a README :)

**Comment by ailispaw** [comment]
Do you mean the main README, a separate one, or both?

**Comment by prologic** [comment]
> Do you mean the main README, a separate one, or both?

The main README is already "too full" and no-one will read another README tucked away in another corner :) -- For the really curious they can "go look" :)

**Comment by ailispaw** [comment]
OK. I will merge run.sh only for now. Thank you. :)

**Comment by prologic** [comment]
We'l see :) "iterate and move fast" :)

**Comment by ailispaw** [comment]
Hmm. It's already merged. Should be closed.

**Pull request closed without merging**